src/ui/StackedBarChart.js

```diff
@@ -911,6 +948,10 @@ WGL.ui.StackedBarChart = function(m, div_id, x_label, filterId, params, category
 
                 parent.append("text")
                     .attr("class", "label out")
+                    .style('text-anchor', 'middle')
+                    .style('fill', 'navy')
+                    .style('font-size','11px')
+                    .style('font-family', '\'Ubuntu\', sans-serif')
                     .attr("x", xOut)
```

Comment:
I see here some code duplicity as the style repeats a couple of times. Can we define that at one place and reuse?

Reply:
Hi Jan, I'd love to keep all those styles organized within a CSS class in a separate file, but to include this charts as they appear in the app in the exported PDFs I needed to write ugly inline styles. Maybe I can search for a cleaner solution to this problem.